CHANGELOG.md

```diff
@@ -0,0 +1,148 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+Releases are available on the [github repository](https://github.com/machines-in-motion/mim_solvers/releases).
+
+## [Unreleased]
+
+
+## [0.1.0] - 2025-03-21
```

Comment:
this line should no appear before the v0.1.0 tag, and it will be added automatically by jrl-cmakemoduls release process